include/great_expectations/expectations/.ge_store_backend_id

```diff
@@ -0,0 +1 @@
+store_backend_id = 28fc8bae-d91c-4ca3-8c9f-b0270ea34e7d
```

Comment:
I need to double-check with the team if this is the best way to set a store_backend_id for a case like this (it might mess with our usage stats).

Reply:
Yeah, I honestly wasn't very clear on what that was used for and didn't notice that it was included in the commits.  If it's for collecting usage stats, then it should be ignored and not committed, I would guess.  Let me know.

Reply:
Removed the usage stats config section completely and ignored the `.ge_store_backend_id` file.